recipes/xmsstamper/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "3.0.4" %}
+
+package:
+  name: xmsstamper
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aquaveo/xmsstamper/archive/{{ version }}.tar.gz
+  sha256: a2ea2c72a0d8f269c0814984ea885674d42b44a5ebe4a396b12ddb45035b8efa
+
+build:
+  number: 0
+  skip: True  # [not py36]
+  merge_build_host: true
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    # just to avoid build/host merge issues
+    # but these should be in host.
+    - zlib
+    - xmscore
+    - xmsgrid
+    - xmsinterp
+    - python
+    - pybind11
+    - boost-cpp
+    - pip
```

Comment:
Most of these should be host deps

Reply:
I worked with @ocefpaf on several other pull requests in this library line, and we were unable to move these to host

Reply:
Yep. There is an issue with conda/conda-build the creates a package conflict when merging host/build. We should also avoid the need to merge them, but that is another problem with the package that won't be addressed now.